### DISCUSSION TITLE
Project Gemini Map 01: Sunken Bar

### DISCUSSION THREAD

**Discussion by Evilslayer506** [nzp-team/nzportable#954] [Map Releases]
This is first map of the "Project: Gemini"* Storyline *(Title Not Final) for Nazi Zombies: Portable where you're trapped in a bar that got caught in a sinkhole and now needs to find whatever it takes to survive...

Created by: Evilslayer506 & Nickbear
Map Design: Evilslayer506
Ideas and Voice Acting: Nickbear

What's in store:
1. Named Survivors (Probably Pointless, but there for some immersion)
2. 3 Easter Eggs (Main Easter Egg with 2 Side Easter Eggs)
3. Optinal Announcer voiced by Nickbear

NOTE: This map only works on PC, doesn't work on 3DS and PS Vita

![fte-20241028212358-0](https://github.com/user-attachments/assets/4bd7ce0b-df57-4732-b15e-9c58cacd10c1)
![fte-20241028212611-0](https://github.com/user-attachments/assets/08f364ca-da45-492c-b861-4c00e0c3cdcd)
![fte-20241028212711-0](https://github.com/user-attachments/assets/df7a7a4c-2cdc-4f88-b857-602f90a171b3)
![fte-20241028212800-0](https://github.com/user-attachments/assets/9fec8aa2-77f4-43b4-bbc3-b9293fa7552c)
![fte-20241028213055-0](https://github.com/user-attachments/assets/ceba572b-e1f2-4b03-92e6-5b83bfaed656)
![fte-20241028213125-0](https://github.com/user-attachments/assets/e6a2fd91-05b8-431c-95f4-4afaff78c1fd)

https://github.com/Evilslayer506/Sunken-Bar/archive/refs/heads/main.zip

**Comment by MotoLegacy** [comment]
> NOTE: This map only works on PC, doesn't work on 3DS and PS Vita

why? let's fix that

**Reply by katniny** [reply]
its a 31mb map uncompressed
the custom announcer goes way past the max file size limit for a map by themselves, not accounting for the radios and actual map itself

**Reply by MotoLegacy** [reply]
yeah, they should fix that lol

**Reply by Evilslayer506** [reply]
> its a 31mb map uncompressed the custom announcer goes way past the max file size limit for a map by themselves, not accounting for the radios and actual map itself

So, What's the max file size limit?

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard because it does not work on all platforms and the download link does not meet spec: https://github.com/nzp-team/nzportable/discussions/159
